src/Chapter1.hs

```diff
@@ -551,8 +558,15 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
-
+isVowel :: Char -> Bool
+isVowel c
+    | c == 'A' || c == 'a' = True
+    | c == 'E' || c == 'e' = True
+    | c == 'I' || c == 'i' = True
+    | c == 'O' || c == 'o' = True
+    | c == 'U' || c == 'u' = True
+    | c == 'Y' || c == 'y' = True
```

Comment:
Nice catch on the uppercase letters!

To be honest we didn't cover them in tests 😃 Thanks for noticing 👁️